core/models/asset.py

```diff
@@ -586,6 +586,11 @@ class StateTransition(models.Model):
         'AssetIncidentReport', on_delete=models.PROTECT
     )
     state = models.CharField(max_length=50, default=constants.NEWLY_REPORTED)
+    state_transition = models.CharField(
```

Comment:
I believe this is covered by the state field.
`Add internal assessment option in transition state field . with options requires repair , requires external assessment, damaged or lost`

Reply:
hmm.. seeing how this might be confusing - the idea is when an incidence report is submitted the admins can transition it from one state to the next.
- newly reported
- internal assessment
  - requires repair
  - requires external assessment
  - damaged
  - lost

i suggest we add a field that identifies whether a state has a 'parent' or not

Reply:
As I understand it, the additional field added is to provide more detail (lost, damaged etc) on submitting a report.
Are you suggesting to add another aside from that? If so, kindly elaborate on 'parent'.

Reply:
we always start with a `newly reported` state. after this the asset undergoes internal assessment the department decides whether it requires external assessment, repair or should be marked as damaged.
in this case the states would be something like:
`newly reported`=> `internal assessment` => `external assessment`=> `out for repair`

the options `requires repair`, `requires external assessment`, `damaged` belong to the `internal assessment` state. we can go with details but the field name should match the functionality.

please discuss this further with @MadgeKinyanjui in case i've missed something